docs/developer/app-store/apps/adyen.mdx

```diff
@@ -0,0 +1,176 @@
+
+The Saleor App Payment Adyen is designed to simplify the payment process for both merchants and customers by seamlessly integrating Adyen's payment processing technology into the Saleor platform. With this app, merchants can easily set up and manage their Adyen account and accept a wide range of payment methods directly from their Saleor store, without needing to integrate Adyen's payment gateway separately.
+
+In addition to processing payments, the Saleor App Payment Adyen provides merchants with tools for managing refunds and chargebacks. Overall, the Saleor App Payment Adyen is a powerful tool for merchants looking to streamline their payment processes and offer their customers a seamless and secure online shopping experience.
+
+:::caution
+
+To successfully configure the Adyen App, you must have an account with [Adyen](https://www.adyen.com).
+
+:::
+
+## Capabilities
+
+The Adyen App implements the following [Saleor sync webhooks related to transactions](../../extending/apps/synchronous-webhooks/transaction-webhooks):
+
+- [`PAYMENT_GATEWAY_INITIALIZE_SESSION`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum#code-style-fontweight-normal-webhookeventtypesyncenumbpayment_gateway_initialize_sessionbcode)
+- [`TRANSACTION_INITIALIZE_SESSION`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum.mdx#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_initialize_sessionbcode)
+- [`TRANSACTION_PROCESS_SESSION`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum.mdx#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_process_sessionbcode)
+- [`TRANSACTION_CHARGE_REQUESTED`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_charge_requestedbcode)
+- [`TRANSACTION_CANCEL_REQUESTED`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_cancelation_requestedbcode)
+- [`TRANSACTION_REFUND_REQUESTED`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_refund_requestedbcode)
+
+Furthermore, it's also prepared to handle [async Adyen webhooks](https://docs.adyen.com/development-resources/webhooks).
+
+Adyen App follows the flow described in detail in the [Saleor Payment App documentation](../../payments#saleor-payment-app).
+
+## Configuration
+
+For Adyen to appear as [available payment gateway](../../checkout#listing-available-payment-gateways), you need to install and configure it in the Saleor Dashboard. You need to obtain the API key from Adyen and paste it into the Adyen App configuration form. Then, a wizard will guide you through the process of configuring the Adyen App, generating the HMAC key, and setting up the webhook.
```

Comment:
I would place here a links to:
a) how to install app from dashboard
b) how to set up the account on adyen side (can be adyne doc link)

Reply:
`you need to install and configure it in the Saleor Dashboard.`
Shouldn't be only `you need to install it in the Saleor via Saleor Dashboard` (or something). The `configuration` is explained one sentence later so I would skip it here.